build_sgx_binary.sh

```diff
@@ -1,15 +1,23 @@
 #!/bin/bash
-WORK_DIR=/project/sawtooth-poet/src
+WORK_DIR=/project/sawtooth-poet2/src
 SGX_DIR=/tmp/sgxsdk
 
 #setting SGX environment
 source $SGX_DIR/environment
 
+#Add proxy settings
```

Comment:
Proxy and aesm can be in an if block if SGX is hw mode

Reply:
Ok